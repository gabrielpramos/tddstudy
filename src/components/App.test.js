import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

const app = shallow(<App />);

it('Renders correctly', () => {
    expect(app).toMatchSnapshot();
});