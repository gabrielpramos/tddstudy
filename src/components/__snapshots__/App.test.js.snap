// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rendears correctly 1`] = `ShallowWrapper {}`;

exports[`Renders correctly 1`] = `ShallowWrapper {}`;

exports[`renders correctly 1`] = `ShallowWrapper {}`;
